src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -306,13 +306,25 @@ export const useSwapper = () => {
           sellAssetAccountNumber: 0, // TODO: remove hard coded accountId when multiple accounts are implemented
           buyAssetAccountNumber: 0, // TODO: remove hard coded accountId when multiple accounts are implemented
           wallet,
-          sendMax: true,
+          sendMax: false,
           receiveAddress,
         })
       } else if (sellAsset.chainId === KnownChainIds.BitcoinMainnet) {
-        // TODO do bitcoin specific trade quote including `bip44Params`, `accountType` and `wallet`
-        // They will need to have selected an accountType from a modal if bitcoin
-        throw new Error('bitcoin unsupported')
+        const { accountType, utxoParams } = getBtcUtxoParams(accountSpecifiersList, sellAsset)
+        if (!utxoParams?.bip44Params) throw new Error('no bip44Params')
+        return swapper.buildTrade({
+          chainId: KnownChainIds.BitcoinMainnet,
```

Comment:
purely a comment - the swapper should be able to infer the `chainId` from the `sellAsset` making this redundant, but let's not change this now

Reply:
yeah was commenting to @gomesalexandre the same thing on lib side